src/main/java/org/jboss/logmanager/ExtHandler.java

```diff
@@ -129,7 +129,7 @@ protected void publishToNestedHandlers(final ExtLogRecord record) {
                                 if (record.getFormatStyle() == ExtLogRecord.FormatStyle.PRINTF) {
                                     // reformat it in a simple way, but only for legacy handler usage
                                     oldRecord = new ExtLogRecord(record);
-                                    oldRecord.setMessage(record.getFormattedMessage());
+                                    oldRecord.setMessage(record.getFormattedMessage(), ExtLogRecord.FormatStyle.NO_FORMAT);
```

Comment:
This will not work. The handler is a legacy handler and thus doesn't understand `NO_FORMAT`. We either have to escape the output or change the behavior (for example, reset the message to `{0}` and put the formatted message into the sole parameter slot).

Reply:
After discussion, the conclusion is that this *will* work. Not because the legacy handler understands `NO_FORMAT`, but because `NO_FORMAT` is actually the behavior that is present for legacy formatters when the parameters array is `null` or empty.